react/components/SearchResult.js

```diff
@@ -130,9 +137,9 @@ export default class SearchResult extends Component {
     }
 
     const hideFacets = !map || !map.length
-
     const showLoading = loading && !fetchMoreLoading
     const showContentLoader = showLoading && !showLoadingAsOverlay
+    const filterClasses = classNames({ 'flex justify-center flex-auto br bl b--muted-4 bw1': mobileMode })
```

Comment:
It is not better to use tachyons media queries? See the [documentation](https://vtex.github.io/vtex-tachyons/) for more details.